draft-tschofenig-rats-psa-token.md

```diff
@@ -592,8 +600,8 @@ For example, an hypothetical profile using only COSE_Mac0 with the AES Message A
 ## Backwards Compatibility Considerations
 {: #sec-backwards-compat}
 
-A previous version of this specification (identified by the `PSA_IOT_PROFILE_1`
-profile) used claim key values from the "private use range" of the CWT Claims
+A previous version of this specification {{PSA-OLD}}, identified by the `PSA_IOT_PROFILE_1`
```

Comment:
```suggestion
A previous version of this specification {{PSA-OLD}} identified by the `PSA_IOT_PROFILE_1`
```

Reply:
This should be right.  It's an _"inciso"_ (in Italian, an "aside" in English) and the two commas setting off the additional info are OK.